src/components/DateInput/PDateButton.vue

```diff
@@ -33,16 +33,26 @@
 
   const props = defineProps<{
     date: Date | null | undefined,
+    showTime?: boolean,
   }>()
 
   const buttonElement = ref<HTMLButtonElement>()
   const el = computed(() => buttonElement.value)
+
+  const dateTimeFormat = computed(() => {
+    if (props.showTime) {
+      return 'MMM do, yyyy \'at\' h:mm a'
+    }
+
+    return 'MMM do, yyyy'
+  })
```

Comment:
Any chance of localization? This is strictly better than the current version but leans in to US style dates and I'm assuming `date-fns` isn't taking that into account when passing a specific format

Reply:
Correct it does not. I briefly looked into that but decided that was more of a change than I wanted to make today. This aligns it with the rest of our products (which also do not consider localization).